src/Dapr.Client/Protos/dapr/proto/dapr/v1/appcallback.proto

```diff
@@ -160,6 +160,9 @@ message TopicSubscription {
   // The optional routing rules to match against. In the gRPC interface, OnTopicEvent
   // is still invoked but the matching path is sent in the TopicEventRequest.
   TopicRoutes routes = 5;
+
+  // The optional dead letter queue for this topic to send events to.
+  string dead_letter_topic = 6;
```

Comment:
Just to check, this is copied from `dapr/dapr`, right?

Reply:
Yes